api/src/main/java/org/eclipse/microprofile/rest/client/RestClientBuilder.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.rest.client;
+
+import javax.annotation.Priority;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ServiceLoader;
+
+public abstract class RestClientBuilder {
+    public static RestClientBuilder newBuilder() {
```

Comment:
I'll prefer newInstance() or instance()

Reply:
`instance()` tends to imply singleton, which isn't the case here.  I'd prefer to keep it more aligned to the actual instance name.  @pilhuhn @andymc12 @kenfinnigan thoughts?

Reply:
I like newBuilder() - it is consistent with the JAX-RS Client API.